src/model/article-vor.v4.yaml

```diff
@@ -2,18 +2,9 @@ $schema: http://json-schema.org/draft-04/schema#
 title: Article VoR
 type: object
 allOf:
-  - $ref: ../common/article-vor.v1.yaml
+  - $ref: ../snippets/article-vor.v1.yaml
```

Comment:
@lsh-0 this is the significant change. The `article-vor.v4` schema not inherits from the whole of the snippet schema. So a valid full article will produce a valid snippet.

Reply:
cheers, @nlisgo 

feel free to merge